README.md

```diff
@@ -62,13 +65,25 @@ The Evervault Node.js SDK exposes four functions.
 `evervault.encrypt()`encrypts data for use in your [Functions](https://docs.evervault.com/tutorial). To encrypt data at the server, simply pass an object or string into the evervault.encrypt() function. Store the encrypted data in your database as normal.
 
 ```javascript
-async evervault.encrypt(data: Object | String);
+async evervault.encrypt(data: Object | String | Buffer);
 ```
 
 | Parameter | Type             | Description           |
 | --------- | ---------------- | --------------------- |
 | data      | Object or String | Data to be encrypted. |
 
+### evervault.decrypt()
+
+`evervault.decrypt()` decrypts the data previously encrypted with the `encrypt()` function or through Relay.
+
+```javascript
+async evervault.decrypt(encrypted: Object | String | Buffer);
```

Comment:
I think we should also support primitive types here for better DX since that's what we support in the encrypt function. I feel it's kinda smelly to only accept Objects.
I solved this by wrapping any non buffer type into a "data" field in a json object.

Reply:
Types accepted by the decrypt function should be String, Array, Object or Buffer!